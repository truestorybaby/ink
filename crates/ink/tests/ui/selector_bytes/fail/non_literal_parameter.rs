const INPUT: &str = "test";
const _: u32 = ink::selector_bytes!(INPUT);

fn main() {}
