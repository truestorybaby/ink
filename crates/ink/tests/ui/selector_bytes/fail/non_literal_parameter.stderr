error: expected string or byte string literal as input: expected literal
 --> tests/ui/selector_bytes/fail/non_literal_parameter.rs:2:37
  |
2 | const _: u32 = ink::selector_bytes!(INPUT);
  |                                     ^^^^^
